domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -402,6 +402,109 @@ public void deleteByIdResponseWithout204(TestContext testContext) {
         asyncAssertSuccess(testContext, 500, "respond204"));
   }
 
+  @Test
+  public void deleteByCQLwithNo500(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    //  insert(testContext, pg, "delete_test",  1);
+    PgUtil.delete("users",  "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout500.class,
+        asyncAssertFail(testContext, "respond500"));
+  }
+
+  @Test
+  public void deleteByCQLwithNo400(TestContext testContext) {
+    PgUtil.delete("users", "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout400.class,
+        asyncAssertSuccess(testContext, 500, "respond400"));
+  }
+
+  @Test
+  public void deleteByCQLwithNo204(TestContext testContext) {
+    PgUtil.delete("users", "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout204.class,
+        asyncAssertSuccess(testContext, 500, "respond204"));
+  }
+
+  @Test
+  public void deleteByCQLNullHeaders(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    insert(testContext, pg, "delete_test",  1);
+    PgUtil.delete("users",  "username==delete_test",
+        null, vertx.getOrCreateContext(), Users.DeleteUsersByUserIdResponse.class,
+        asyncAssertSuccess(testContext, 400, "null"));
+  }
+
+  @Test
+  public void deleteByCQLOK(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    insert(testContext, pg, "delete_test",  1);
+    String cql = "username=delete_test";
```

Comment:
This doesn't test the key feature of this method: Deleting more than one record at a time.
Why don't you create "delete_a", "delete_b1", "delete_b2" and delete by CQL "username=delete_b*"?

Reply:
Done, though I'm really not too worried .. PostgresClient.delete does the job.